development/build/utils.js

```diff
@@ -0,0 +1,21 @@
+function getNextBetaVersionMap(currentVersion, platforms) {
```

Comment:
This function is fairly naive in that it presumes this is `premajor 0`, so a second beta of the same version would have its premajor set to 0 as well; we'll need to be clever here.  We could use a `semver` package but that might just be another dependency.
2.